client/src/components/NavBar/BecomeSitter/BecomeSitter.tsx

```diff
@@ -0,0 +1,108 @@
+import useStyles from './useStyles';
+import { ChangeEvent, Fragment, useState } from 'react';
+import Button from '@material-ui/core/Button';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import FormControl from '@material-ui/core/FormControl';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import Switch from '@material-ui/core/Switch';
+import { Checkbox, FormGroup, TextField, Typography } from '@material-ui/core';
+import Box from '@material-ui/core/Box';
+
+const BecomeSitter = (): JSX.Element => {
+  const classes = useStyles();
+
+  const [state, setState] = useState({
+    mon: false,
+    tue: false,
+    wed: false,
+    thurs: false,
+    fri: false,
+    sat: false,
+    sun: false,
+  });
+
+  const [open, setOpen] = useState(false);
+  const [isSitter, setIsSitter] = useState(false);
+
+  const handleClickOpen = () => {
+    setOpen(true);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
+  const handleChange = (event: ChangeEvent<HTMLInputElement>) => {
+    setState({
+      ...state,
+      [event.target.name]: event.target.checked,
+    });
+  };
+
+  const handleSitter = (event: ChangeEvent<HTMLInputElement>) => {
+    setIsSitter(event.target.checked);
+  };
+
+  const handleSubmit = () => {
+    //TODO: Placeholder for editing profile info to switch between sitter and not a sitter
+    console.log('hello');
```

Comment:
Also, remove this console log - :smiley_cat:

Reply:
removed.